commerce-test-util/bnd.bnd

```diff
@@ -1,4 +1,4 @@
 Bundle-Name: Liferay Commerce Test Utilities
 Bundle-SymbolicName: com.liferay.commerce.test.util
-Bundle-Version: 4.1.2
+Bundle-Version: 4.2.0
```

Comment:
@riccardo-alberti can you make these changes to `bnd.bnd` and `packageinfo` in a separate commit, just for semantic versioning? These lines are likely to conflict, since other pulls may also make updates to this module and thus require baseline changes, so having a separate commit for the autogenerated changes makes conflict resolution really easy (i.e. just wipe the changes and regenerate them).